packages/admin-panel/src/index.js

```diff
@@ -1,33 +1,34 @@
-/**
- * Tupaia MediTrak
- * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
-
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
 import { render as renderReactApp } from 'react-dom';
-import { MuiThemeProvider, StylesProvider } from '@material-ui/core/styles';
-import { ThemeProvider } from 'styled-components';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { Provider } from 'react-redux';
-import { PersistGate } from 'redux-persist/lib/integration/react';
 import 'react-table/react-table.css';
+import AdminPanel from './App';
+import { AdminPanelProviders, VizBuilderProviders } from './utilities';
+import { Footer, Navbar } from './widgets';
 
-import { theme } from './theme';
-import { store, persistor } from './store';
-import { App } from './App';
+const VizBuilder = lazy(() => import('./vizBuilder'));
```

Comment:
Lazy loading the viz builder means that webpack will split it into its own bundle and only serve it to the user when they hit the `viz-builder` route